src/app/header/menu/menu.component.scss

```diff
@@ -162,16 +167,56 @@ ul {
             &[href]:hover {
               color: $color-text;
             }
+
+            a {
+              padding: 0;
+            }
+
+            button {
+              background: $color-accent;
+              border: none;
+              color: $white;
+              font-size: 1.4rem;
+              font-weight: bold;
+              height: 1.4rem;
+              line-height: 1.4rem;
+              padding: 0;
+              text-align: center;
+              width: 1.4rem;
+            }
           }
 
           .submenu-section {
             @include list-styling;
+            display: none;
+            flex-wrap: wrap;
             line-height: 1em;
-            // padding: 0 0 0 1em;
             padding: 0;
 
+            &.open {
+              border-top: 1px solid $color-accent;
+              display: flex;
+              height: 100%;
+              overflow-y: auto;
+            }
+
+            &.wrapped {
+              li {
+                align-content: center;
+                align-items: center;
+                display: flex;
+                max-height: 50vh;
```

Comment:
I think this max height is in wrong place. 
Now that the menu is accordion I think max-height should be defined in outer element / border element. Then if menu would grow too large, whole menu would scroll not just sub items.